src/components/SmallButton.stories.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import SmallButton from './SmallButton'
+
+export default {
+  title: 'SmallButton',
+  component: SmallButton,
+}
+
+const Template = args => <SmallButton {...args} />
+
+export const Default = Template.bind({})
+Default.args = {
+  children: '-',
+}
```

Comment:
Maybe a 2nd option with props for right and top?

Reply:
it has no parent element in storybook I need to look into this how it works in storybook...

Reply:
Solved it